src/QuadForm/Lattices.jl

```diff
@@ -1471,20 +1489,21 @@ end
 #
 ################################################################################
 
-# bugged needs intersections of non-full modules to work first
 @doc Markdown.doc"""
-    _orthogonal_complement(M::AbsLat, L::AbsLat) -> AbsLat
+    orthogonal_submodule(L::AbsLat, M::AbsLat) -> AbsLat
 
-Return the orthogonal complement of the lattice `L` inside the lattice `M`.
+Return the largest submodule of `L` orthogonal to `M`.
 """
-function _orthogonal_complement(M::AbsLat, L::AbsLat)
-  @req ambient_space(M) == ambient_space(L) "lattices must be in the same ambient space"
+orthogonal_submodule(L::AbsLat, M::AbsLat) = orthogonal_submodule(L, M)
```

Comment:
Hm, what is this method for?

Reply:
Is this just some leftover?

Reply:
Ah no it is to get the documentation correctly called in the file for the html documentation, so that I can leave the signatures `AbsLat` while for the method I use ` T where T <: Union{HermLat, QuadLat}`